buildsystem/HandlePythonOptions.cmake

```
@@ -4,7 +4,17 @@
 
 # the Python version number requirement is in modules/FindPython_test.cpp
 find_package(Python ${PYTHON_MIN_VERSION} REQUIRED)
-find_package(Cython ${CYTHON_MIN_VERSION} REQUIRED)
+find_package(Cython 3.0.10 QUIET)
+if (CYTHON_FOUND AND CYTHON_VERSION VERSION_GREATER_EQUAL 3.0.10)
+    message("Using Cython >= 3.0.10 (found version ${CYTHON_VERSION})")
```

```suggestion
if (CYTHON_FOUND)
    message("Using Cython >= ${CYTHON_MIN_VERSION} (found version ${CYTHON_VERSION})")
```

You don't need the second condition in the if as that is already checked by line 7. We should also still try to use the `CYTHON_MIN_VERSION` parameter.

---

Fixed